scripts/prettier/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+// Based on similar script in Jest
+// https://github.com/facebook/jest/blob/master/scripts/prettier.js
+
+const chalk = require('chalk');
+const glob = require('glob');
+const path = require('path');
+const execFileSync = require('child_process').execFileSync;
+
+const shouldWrite = process.argv[2] === 'write';
+const isWindows = process.platform === 'win32';
+const prettier = isWindows ? 'prettier.cmd' : 'prettier';
+const prettierCmd = path.resolve(__dirname, '../../node_modules/.bin/' + prettier);
+const defaultOptions = {
+  'bracket-spacing': 'false',
+  'single-quote': 'true',
+  'jsx-bracket-same-line': 'true',
+  'trailing-comma': 'all',
+  'print-width': 80,
+};
+const config = {
+  default: {
+    patterns: ['src/**/*.js'],
+    ignore: [
+      '**/third_party/**',
+      '**/node_modules/**',
+    ],
+  },
+};
+
+function exec(command, args) {
+  console.log('> ' + [command].concat(args).join(' '));
```

Comment:
I find this barely more helpful without the .join but if you like this then we can keep it.